src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$stars-size: 16px;
+
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+  font-weight: 500;
+}
+
+.container {
+  --column-count: 4;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+/* ====== Header ====== */
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 3px 0 0;
+}
+
+.menu {
+  font-size: 12px;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.link {
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  display: flex;
+}
```

Comment:
The `.link` class has duplicate styles with `.is-active` (lines 82-90). To follow DRY (Don't Repeat Yourself) principles, consider combining shared styles and using modifiers only for the differences.